src/vs/workbench/contrib/tasks/browser/abstractTaskService.ts

```diff
@@ -2555,9 +2595,16 @@ export abstract class AbstractTaskService extends Disposable implements ITaskSer
 
 		ProblemMatcherRegistry.onReady().then(() => {
 			return this.editorService.saveAll({ reason: SaveReason.AUTO }).then(() => { // make sure all dirty editors are saved
+				// {{SQL CARBON EDIT}}
+				let lastRunTasksViewTask = this.lastRunTasksViewTask;
```

Comment:
Why is this necessary? Shouldn't the rerun function below be the one doing this kind of thing?

Reply:
The ADS Tasks system is different from the VS Code Task system, so to include it in the Tasks View, I had to create another variable.